### DISCUSSION TITLE
#72 日本东京NTT-1线路 国内墙了

### DISCUSSION THREAD

**Discussion by awmc233** [natfrp/wiki#141] [使用问题]
#72 日本东京NTT-1线路 国内已经被墙了，加速到HK就能用
![image](https://user-images.githubusercontent.com/87456168/184719441-72228659-b8ff-4d6f-b8d5-8be9af4c863c.png)
![image](https://user-images.githubusercontent.com/87456168/184719526-e194e0e2-e437-4e40-828c-289fe38afd6a.png)
![image](https://user-images.githubusercontent.com/87456168/184720298-512c0cf1-6bec-4055-a901-c804246b0854.png)

**Comment by awmc233** [comment]
我知道东京线路建议海外用户用，但是没有备案真的很烦，懒得备案（听说还要线下找工信部的人拍照，我这根本没有）

**Reply by awmc233** [reply]
现在我在墙内
![image](https://user-images.githubusercontent.com/87456168/184721730-72315683-3326-4946-99b6-777b0e4855a2.png)
不知道为啥，延迟检测过了

**Reply by awmc233** [reply]
![image](https://user-images.githubusercontent.com/87456168/184722559-74bd7c51-49a9-4a89-ba28-daafafa86f94.png)
复活了，但是又完全没复活

**Comment by iamckun** [comment]
挂了代理也连不上了，感觉像端口没开一样，可以在浏览器打开这个ip的地址
